qiskit/algorithms/gradients/lin_comb_qgt.py

```diff
@@ -0,0 +1,292 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+A class for the Quantum Fisher Information.
+"""
+
+from __future__ import annotations
+
+from copy import copy
+from typing import Sequence
+
+import numpy as np
+
+from qiskit.algorithms import AlgorithmError
+from qiskit.circuit import Parameter, QuantumCircuit
+from qiskit.primitives import BaseEstimator
+from qiskit.primitives.utils import _circuit_key
+from qiskit.providers import Options
+from qiskit.quantum_info import SparsePauliOp
+
+from .base_qgt import BaseQGT
+from .lin_comb_estimator_gradient import LinCombEstimatorGradient
+from .qgt_result import QGTResult
+from .utils import  DerivativeType, _make_lin_comb_qgt_circuit, _make_lin_comb_observables
+
+
+class LinCombQGT(BaseQGT):
+    """Computes the Quantum Fisher Information (QFI) given a pure,
+    parameterized quantum state. This method employs a linear
+    combination of unitaries [1].
+
+    **Reference:**
+    [1] Schuld et al., Evaluating analytic gradients on quantum hardware, 2018
```

Comment:
Does this reference apply here? or is it for QFI?

Reply:
It' apply for the linear combination gradient (or QGT).